maps/Route36.asm

```diff
@@ -48,6 +48,7 @@ Route36_MapScriptHeader:
 	const ROUTE36_COOLTRAINERF
 
 Route36ArthurCallback:
+	callasm UpdatePlayerTallGrassFlags
```

Comment:
Can/should this just be done unconditionally when you exit any grotto?

Reply:
Depends on what you mean... You can create an object callback with that line on every map that has a grotto...

Or... I could include it as a mapscript... and it would do it upon *every* map load.. Which would work, and *shouldn't* cause any problem... but I was trying to avoid make a wide spread change for what appeared to be a single instance of a problem. Would you prefer the mapscript option?

Reply:
A mapscript would be fine. I think exiting a grotto could set a flag to true; then the new mapscript would check this flag, just return if it's false, but if it's true then do the fix and set the flag to false.